cocos/core/data/class.ts

```diff
@@ -594,6 +593,7 @@ function parseAttributes (constructor: Function, attributes: PropertyStash, clas
     parseSimpleAttribute('min', 'number');
     parseSimpleAttribute('max', 'number');
     parseSimpleAttribute('step', 'number');
+    parseSimpleAttribute('userData', 'object');
```

Comment:
In the old logic, `userData` was exposed only in DEV mode. 
Why does it need to be exposed in Release mode too?

Reply:
Well, in that case, I don't necessarily need him, I just happened to see such a field in the Property decorator, and from the comments, it doesn't seem to be limited to use under editor or dev. I'm just adding some class tolerance for my properties to do something later. Both so I can develop a new decorator to do this.

Reply:
So will you reverse the `userData` changes?